src/containers/MyNdla/components/UserProfileTag.tsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import styled from '@emotion/styled';
+import { colors, spacing, misc } from '@ndla/core';
+import SafeLink from '@ndla/safelink';
+import { useTranslation } from 'react-i18next';
+import { Text } from '@ndla/typography';
+import { checkIfModerator } from '../../../config';
+import Avatar from './Avatar';
+import { useArenaUser } from '../arenaQueries';
```

Comment:
Eg liker denne rekkefølgen på ting. Eksterne imports, @ imports og så interne imports. Synes det er ryddigere.
```suggestion
import { useTranslation } from 'react-i18next';
import styled from '@emotion/styled';
import { colors, spacing, misc } from '@ndla/core';
import SafeLink from '@ndla/safelink';
import { Text } from '@ndla/typography';
import { checkIfModerator } from '../../../config';
import Avatar from './Avatar';
import { useArenaUser } from '../arenaQueries';
```